src/stories/UserCanCreateQueries/story.js

```diff
@@ -0,0 +1,63 @@
+const validator = requireValidator();
+const QueryRepo = requireRepo("query");
+const QuerySerializer = requireSerializer("query");
+const findKeysFromRequest = requireUtil("findKeysFromRequest");
+
+const prepare = ({ reqQuery, reqBody, reqParams, req }) => {
+  const payload = findKeysFromRequest(req, ["title", "anonymous"]);
+  console.log(req.user,"useruser")
+  payload["owner_uuid"] = "01"
```

Comment:
Have to discuss on where this would be coming from, but for now u can use `req.user`

Reply:
ok i tried req.user but it was coming undefined